Scones/glob.py

```diff
@@ -0,0 +1,34 @@
+def glob(env, root: str, extensions: list):
```

Comment:
This is *very* close to the builtin `Glob` function. Perhaps we could call ours `RecursiveGlob`?

Reply:
Addressed in a9bfd261b36f4f272ff1412e79cecbe6b2258d22